autogpt_platform/frontend/src/components/layout/Navbar/components/Wallet/Wallet.tsx

```diff
@@ -339,8 +339,8 @@ export function Wallet() {
               <span className="text-sm font-semibold">
                 {formatCredits(credits)}
               </span>
-              {completedCount && completedCount < totalCount && (
-                <span className="absolute right-1 top-1 h-2 w-2 rounded-full bg-violet-600"></span>
+              {completedCount !== null && completedCount < totalCount && (
```

Comment:
`completedCount` and `completedCount !== null` should have the same effect here?